src/js/actions/export.js

```diff
@@ -374,9 +374,15 @@ define(function (require, exports) {
      * Adds an asset, or assets to the end of the document asset list, or that of a layer or layers
      * If props not provided, choose the next reasonable scale and create an otherwise vanilla asset
      *
+     * Recognizes an empty list of layers as implying doc-level export
```

Comment:
yes, I know what you mean.  Although I tried to minimize the places that uses this ... and I think it makes it easier here for React to just shovel the selected layers in here and let the action understand "empty selected" .